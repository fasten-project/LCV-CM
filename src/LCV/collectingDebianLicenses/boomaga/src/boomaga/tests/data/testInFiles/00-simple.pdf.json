{
    "checksum": "45532278b5d49cac9b2ef03118b827c7fed3bac16e184c567e97b4bacc202281",
    "file": "00-simple.pdf",
    "mime": {
        "encoding": "iso-8859-1",
        "type": "application/pdf"
    },
    "number_of_duplicates": 1,
    "package": "boomaga",
    "path": "boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/00-simple.pdf",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/boomaga/3.0.0-1/",
        "metric": {
            "size": 5604
        },
        "pts_link": "https://tracker.debian.org/pkg/boomaga",
        "sloc": [
            [
                "cpp",
                13687
            ],
            [
                "sh",
                314
            ],
            [
                "xml",
                9
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://gitlab.com/perezmeyer/debian-package-boomaga",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/00-simple.pdf",
    "stat": {
        "perms": "rw-r--r--",
        "size": 810,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": false,
    "type": "file",
    "version": "3.0.0-1"
}