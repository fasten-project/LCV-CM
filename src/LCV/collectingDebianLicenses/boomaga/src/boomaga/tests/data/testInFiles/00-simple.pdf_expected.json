{
    "checksum": "75a8a6fa04ba6348afdf9f0f786c31b2243d894c8f24799d368dda4da0821133",
    "file": "00-simple.pdf_expected",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 1,
    "package": "boomaga",
    "path": "boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/00-simple.pdf_expected",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/boomaga/3.0.0-1/",
        "metric": {
            "size": 5604
        },
        "pts_link": "https://tracker.debian.org/pkg/boomaga",
        "sloc": [
            [
                "cpp",
                13687
            ],
            [
                "sh",
                314
            ],
            [
                "xml",
                9
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://gitlab.com/perezmeyer/debian-package-boomaga",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/00-simple.pdf_expected",
    "stat": {
        "perms": "rw-r--r--",
        "size": 48,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "3.0.0-1"
}